src/ip/Address.h

```diff
@@ -51,7 +51,6 @@ class Address
     Address(const struct hostent &);
     Address(const struct addrinfo &);
     Address(const char*);
-    ~Address() {}
```

Comment:
FYI: This removal is related to get/putPod() checks discussed in another comment in this review. This removal is explained in branch commit 6455b62:

> Ip::Address becomes a POD from C++ point of view if we remove the unnecessary custom (but empty) destructor. That destructor was also violating the C++ Rule Of Three.